content/en/docs/tasks/configure-pod-container/configure-volume-storage.md

```diff
@@ -71,6 +71,8 @@ the Pod:
 1. In your shell, list the running processes:
 
     ```shell
+    root@redis:/data/redis# apt-get update
+    root@redis:/data/redis# apt-get install procps
```

Comment:
these commands only works on ubuntu ... so I'd suggest we add some texts instead.

Reply:
@lanchongyizu, thanks for the clarification.